Common/include/CConfig.hpp

```diff
@@ -1224,7 +1224,12 @@ class CConfig {
   unsigned short nSpecies_Init;    /*!< \brief Number of entries of SPECIES_INIT */
 
   /*--- Additional flamelet solver options ---*/
-  su2double flame_init[8];       /*!< \brief Initial solution parameters for flamelet solver.*/
+  FLAMELET_INIT_TYPE flame_init_type = FLAMELET_INIT_TYPE::NONE; /*!< \brief Method for solution ignition for flamelet problems. */
+  su2double flame_init[8];       /*!< \brief Flame front initialization parameters. */
+  su2double spark_init[6];       /*!< \brief Spark ignition initialization parameters. */
+  su2double* spark_reaction_rates; /*!< \brief Source terms for flamelet spark ignition option. */
+  unsigned short nspark;           /*!< \brief Number of source terms for spark initialization. */
+  bool preferential_diffusion = false;  /*!< \brief Preferential diffusion physics for flamelet solver.*/
```

Comment:
It's good that users do not depend on the python wrapper for initializing a flame, but in many applications more complex initialization is necessary. Let's put 'adding a python wrapper example' on the to do list.